doc/can-query-logic.md

```diff
@@ -432,52 +506,37 @@ For example, you might want to represent a date with a string like:
 
 The following creates a `DateStringSet` that translates a date string to a number:
 
-```js
-class DateStringSet {
-    constructor(value){
-        this.value = value;
-    }
-    // used to convert to a number
-    valueOf(){
-        return new Date(this.value).getTime()
-    }
-}
-```
+@sourceref ./can-query-logic-models/date-string-example.js
+@codepen
```

Comment:
I'd add some inline docs around `"can.serialize"` to explain what that is for ....

Reply:
And around `[Symbol.for("can.SetType")]: DateStringSet`